CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 <!-- ## [Unreleased] -->
 
+## [5.1.1] - 2018-07-09
```

Comment:
Please reference this tag at the bottom of the CHANGELOG as well

Reply:
Done.